# build fails with:
# /usr/lib/i386-linux-gnu/libc_nonshared.a(stack_chk_fail_local.oS): In function `__stack_chk_fail_local':
# (.text+0x10): undefined reference to `__stack_chk_fail'
#
# at least on i386
Index: gemrb-0.8.2/CMakeLists.txt
===================================================================
--- gemrb-0.8.2.orig/CMakeLists.txt	2015-01-02 22:47:54.055731472 +0100
+++ gemrb-0.8.2/CMakeLists.txt	2015-01-02 22:47:54.051731472 +0100
@@ -372,6 +372,7 @@
 		elseif (CMAKE_SYSTEM_NAME STREQUAL "Haiku")
 			TARGET_LINK_LIBRARIES(${plugin_name} -lbsd)
 		endif (CMAKE_SYSTEM_NAME STREQUAL "NetBSD")
+		ADD_DEFINITIONS("-fno-stack-protector")
 		INSTALL(TARGETS ${plugin_name} DESTINATION ${PLUGIN_DIR})
 	endif (STATIC_LINK)
 
